apps/web-giddh/src/app/subscription/subscription.component.html

```diff
@@ -1,353 +1,382 @@
+                                    <input-field
+                                        [label]="localeData?.auth_key?.main_auth_key"
+                                        #authKeyUrl
+                                        [id]="'authKey'"
+                                        [name]="'userAuthKey'"
+                                        [(ngModel)]="userAuthKey"
+                                        [readonly]="true"
+                                    ></input-field>
+                                </div>
+                                <button type="button" mat-stroked-button color="primary" (click)="toggleIsCopied()">
+                                    {{ commonLocaleData?.app_copy }}
+                                </button>
+                            </div>
+                            <div class="row mb-3" *ngIf="isCopied">
+                                <div class="col-12 col-sm-12 col-md-12 col-lg-6">
+                                    <span class="success">{{ localeData?.auth_key?.copied }}</span>
+                                </div>
+                            </div>
+                            <div class="form-group clearfix mb-0">
+                                <button
+                                    mat-stroked-button
+                                    color="primary"
+                                    type="submit"
+                                    (click)="regenerateKey()"
+                                    class="mr-4"
+                                >
+                                    {{ localeData?.auth_key?.regenerate_auth_key }}
+                                </button>
+                                <a class="text-primary" href="{{ apiPostmanDocUrl }}" target="_blank">{{
+                                    localeData?.auth_key?.documentation
+                                }}</a>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </mat-tab>
+            <mat-tab [label]="localeData?.mobile_number?.tab_heading">
+                <div class="wrap-tab-content-pd-15">
+                    <div class="box clearfix mt-1">
+                        <form class="row" name="mobileForm" novalidate>
+                            <div class="col-lg-3 pd-r05">
+                                <input-field
+                                    [label]="localeData?.mobile_number?.add_edit_mobile_number"
+                                    [name]="'phoneNumber'"
+                                    class="max-width-350"
+                                    [allowDecimalDigitsOnly]="true"
+                                    [maxlength]="10"
+                                    [type]="'text'"
+                                    [placeholder]="localeData?.mobile_number?.verification_code_placeholder"
+                                    [(ngModel)]="phoneNumber"
```

Comment:
check what to do for this Ng model

Reply:
do not change anything on old subscription module